.github/workflows/matomo-tests.yml

```
@@ -0,0 +1,238 @@
+name: Matomo Tests
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  actions: read
+  checks: none
+  contents: read
+  deployments: none
+  issues: write
+  packages: none
+  pull-requests: write
+  repository-projects: none
+  security-events: none
+  statuses: none
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  PHP:
+    runs-on: ubuntu-20.04
+    services:
+      redis:
+        image: redis:5.0
+        ports:
+          - 6379:6379
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      mariadb:
+        image: mariadb:latest
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_DATABASE: piwik_tests
```

We should directly use `matomo` instead of `piwik` where possible
```suggestion
          MYSQL_DATABASE: matomo_tests
```